doc/dox_comments/header_files/ecc.h

```diff
@@ -541,6 +563,52 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
 WOLFSSL_API
 int wc_ecc_init(ecc_key* key);
 
+/*!
+    \ingroup ECC
+
+    \brief This function initializes an ecc_key object for future
+    use with message verification or key negotiation.
+
+    \return 0 Returned upon successfully initializing the ecc_key object
+    \return MEMORY_E Returned if there is an error allocating memory
+
+    \param key pointer to the ecc_key object to initialize
```

Comment:
Please include "\param" notation for all arguments.  I see "key", but what about "heap" and "devId"?